receptai.api/Dtos/Recipe/CreateRecipeRequestDto.cs

```diff
@@ -8,22 +8,21 @@ public class CreateRecipeRequestDto
     [MaxLength(255, ErrorMessage = "Title may not be longer than 255 characters!")]
     public string Title { get; set; } = null!;
 
-    public int? ImgId { get; set; }
-
     [Required]
     public int SubfoodditId { get; set; }
 
+    public IFormFile? Photo { get; set; }
+
     [Required]
     public string Ingredients { get; set; } = null!;
 
     public string? CookingTime { get; set; } = null!;
 
+    [Required]
```

Comment:
We should really talk about these in a standup or refinement session. It's quite subjective to say what is a-must to specify when creating a recipe. Current model was based purely on my judgement that ingredients and instructions must be given with other fields (such cooking time or difficulty) ommited.

Reply:
The Range(1, ...) on this & CookingDifficulty makes it into a "required" field from model state perspective, however it's not reflected in Swagger unless [Required] tag appears. So for consistency that's why I added [Required] fields.

But yes, I agree that we can re-evaluate what's actually must-have when creating or updating recipe